tools/nebula-flink/src/main/java/org.apache.flink/connector/nebula/sink/NebulaRowOutputFormatConverter.java

```diff
@@ -0,0 +1,92 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package org.apache.flink.connector.nebula.sink;
+
+import com.esotericsoftware.minlog.Log;
+import org.apache.flink.connector.nebula.statement.ExecutionOptions;
+import org.apache.flink.connector.nebula.utils.NebulaConstant;
+import org.apache.flink.connector.nebula.utils.PolicyEnum;
+import org.apache.flink.types.Row;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NebulaRowOutputFormatConverter implements NebulaOutputFormatConverter<Row> {
+
+    private static final long serialVersionUID = -7728344698410737677L;
+
+    private final int idIndex;
+    private final int srcIdIndex;
+    private final int dstIdIndex;
+    private final int rankIndex;
+
+    public NebulaRowOutputFormatConverter(ExecutionOptions executionOptions) {
+        this.idIndex = executionOptions.getIdIndex();
+        this.srcIdIndex = executionOptions.getSrcIndex();
+        this.dstIdIndex = executionOptions.getDstIndex();
+        this.rankIndex = executionOptions.getRankIndex();
+    }
+
+    /**
+     * convert row to nebula's insert values
+     *
+     * @param row       flink row
+     * @param isVertex  true if row represents vertex
+     * @param policy    see {@link PolicyEnum}
+     *
+     * @return String
+     */
+    @Override
+    public String createValue(Row row, Boolean isVertex, PolicyEnum policy) {
+        if(row == null|| row.equals("")){
```

Comment:
Row compare with an empty String ?

Reply:
wrong code, change it to  row.getArity() == 0